src/js/views/admin/sidebar/clinician/clinician-sidebar_views.js

```diff
@@ -91,79 +126,163 @@ const SidebarView = View.extend({
+      error,
+    }));
+  },
+  showEmail(error) {
+    this.showChildView('email', new EmailView({
+      model: this.clonedClinician,
+      error,
+    }));
   },
   showAccess() {
-    const isDisabled = this.model.isNew();
-    const accessComponent = new AccessComponent({ model: this.model, state: { isDisabled } });
+    const isDisabled = this.clinician.isNew();
+    const accessComponent = new AccessComponent({ model: this.clinician, state: { isDisabled } });
 
     this.listenTo(accessComponent, 'change:access', accessType => {
-      this.model.save({ access: accessType });
+      this.clinician.save({ access: accessType });
     });
 
     this.showChildView('access', accessComponent);
   },
   showRole() {
-    const isDisabled = this.model.isNew();
-    const roleComponent = new RoleComponent({ model: this.model, state: { isDisabled } });
+    const isDisabled = this.clinician.isNew();
+    const roleComponent = new RoleComponent({ model: this.clinician, state: { isDisabled } });
 
     this.listenTo(roleComponent, 'change:role', role => {
-      this.model.saveRole(role);
+      this.clinician.saveRole(role);
     });
 
     this.showChildView('role', roleComponent);
   },
   showGroups() {
-    this.showChildView('groups', new GroupsComponent({
-      clinician: this.model,
-    }));
+    this.showChildView('groups', new GroupsComponent({ clinician: this.clinician }));
   },
   showSave() {
-    if (!this.model.isValid()) {
+    if (!this.clinician.isEditable || !this.clonedClinician.hasChanged()) {
+      this.getRegion('save').empty();
+      return;
+    }
+
+    if (!this.clonedClinician.isValid() || !_.isEmpty(this.errors)) {
       this.showDisabledSave();
       return;
     }
 
-    this.showChildView('save', new SaveView({ model: this.model }));
+    this.showChildView('save', new SaveView({ model: this.clonedClinician }));
   },
   showDisabledSave() {
     this.showChildView('save', new DisabledSaveView());
   },
   showForm() {
-    if (this.model.isNew()) {
-      this.listenTo(this.model, 'change:name change:email', this.showSave);
-      this.showDisabledSave();
+    if (_.isEmpty(this.errors)) {
+      this.cloneClinician();
+    }
+
+    if (this.clinician.isEditable()) {
+      this.listenTo(this.clonedClinician, {
+        'change:name'() {
+          if (this.errors) {
+            this.errors = _.omit(this.errors, 'name');
+          }
+          this.showSave();
+        },
+        'change:email'() {
+          if (this.errors) {
+            this.errors = _.omit(this.errors, 'email');
+          }
+          this.showSave();
+        },
+      });
+      this.showSave();
     }
 
     this.showName();
     this.showEmail();
   },
   showInfo() {
-    if (this.model.isNew()) return;
+    if (this.clinician.isNew()) return;
 
-    if (!this.model.get('_role') || this.model.getGroups().length === 0) {
+    if (!this.clinician.get('_role') || this.clinician.getGroups().length === 0) {
```

Comment:
interestingly I think this could be `if (!this.clinician.isActive())` and maybe should be, but that function is in my other branch.

Reply:
Want me to add it to this PR?

Reply:
it'll be easier to come back to it probably.